types/index.d.ts

```diff
@@ -1,51 +1,11 @@
-import './vue';
-import { PluginFunction } from 'vue/types/plugin';
+import './vue'
+import { VueMeta } from './vue-meta'
 
-/**
- * Installation function
- *
- * @param {Vue} Vue - the Vue constructor.
- * @param {{
- *   keyName: string, // the component option name that vue-meta looks for meta info on.
- *   attribute: string, // the attribute name vue-meta adds to the tags it observes
- *   ssrAttribute: string, // the attribute name that lets vue-meta know that meta info has already been server-rendered
- *   tagIDKeyName: string // the property name that vue-meta uses to determine whether to overwrite or append a tag
- * }} options
- */
-declare const Meta: PluginFunction<{
-  keyName: string, // the component option name that vue-meta looks for meta info on.
-  attribute: string, // the attribute name vue-meta adds to the tags it observes
-  ssrAttribute: string, // the attribute name that lets vue-meta know that meta info has already been server-rendered
-  tagIDKeyName: string // the property name that vue-meta uses to determine whether to overwrite or append a tag
-}>;
+export default VueMeta
 
-export default Meta;
-
-export interface MetaInfo {
-  title?: string
-  titleTemplate?: string | ((titleChunk: string) => string)
-  htmlAttrs?: { [key: string]: string }
-  bodyAttrs?: { [key: string]: string }
-  base?: { target: string, href: string }
-
-  meta?: {
-    vmid?: string,
-    charset?: string,
-    content?: string,
-    'http-equiv'?: 'content-security-policy' | 'refresh',
-    name?: string,
-      [key: string]: any
-  }[]
-
-  link?: { rel: string, href: string, [key: string]: any }[]
-  style?: { cssText: string, type: string, [key: string]: any }[]
-  script?: { innerHTML?: string, src?: string, type: string, [key: string]: any }[]
-  noscript?: { innerHTML: string, [key: string]: any }[]
-
-  __dangerouslyDisableSanitizers?: string[]
-  __dangerouslyDisableSanitizersByTagID?: string[]
-
-  changed?: <T extends object>(newInfo: T, addedTags: HTMLElement[], removedTags: HTMLElement[]) => void
-  afterNavigation?: <T extends object>(vm: Vue, newInfo: T) => void
-  refreshOnceOnNavigation?: boolean
-}
+export {
+  VueMetaOptions,
+  VueMetaPlugin,
+  MetaInfo,
+  MetaInfoSSR
+} from './vue-meta'
```

Comment:
Is there any way to have only once `from './vue-meta'` in this file ?
Maybe should we import everything at the second line and remove the `from` statement here

Reply:
To be honest I think this syntax is a bit cleaner then having to repeat those 4 exports. Does it has any benefits to include a module only once?

Eg https://github.com/vuejs/vue-router/blob/dev/types/index.d.ts does this too..

Reply:
@kevinmarrec Oh well nvm, keep it as it is then :D